pkg/operator/ceph/csi/spec.go

```diff
@@ -136,6 +137,9 @@ var (
 	DefaultResizerImage     = "registry.k8s.io/sig-storage/csi-resizer:v1.5.0"
 	DefaultCSIAddonsImage   = "quay.io/csiaddons/k8s-sidecar:v0.5.0"
 
+	// image pull policy
+	DefaultCSIImagePullPolicy = "IfNotPresent"
```

Comment:
instead of string use `v1.PullIfNotPresent` or do string conversion to avoid human errors?

Reply:
marked as resolved but i dont see any change

Reply:
`(k8sutil.GetValue(data map[string]string, settingName string, defaultValue string)`
^^ defaultValue is expected to be  string, so I set DefaultCSIImagePullPolicy to string

Reply:
what about this?
```suggestion
	DefaultCSIImagePullPolicy = string(v1.PullIfNotPresent)
```